### ISSUE TITLE
Verbose message issue

### ISSUE THREAD

**Issue by paulaten** [erdogant/bnlearn#76]
Hello Erdogant! It's a pleasure to know your work. I have noticed in my latest executions of bnlearn.structure_learning.fit that although I indicate verbose=0, I still get informational messages on the console, specifically: "[bnlearn] >Compute structure scores ['k2', 'bds', 'bic', 'bdeu'] for model comparison (higher is better)."

I have analyzed the code and in the bnlearn/structure_learning.py module, line 194: 
```out['structure_scores'] = bnlearn.structure_scores(out, df)```
, the verbose information is lost and by default, the call to structure_scores has verbose=3. Would it be possible to fix it by adding ```out['structure_scores'] = bnlearn.structure_scores(out, df, verbose=config['verbose'])``` ?

The same thing happens in the bnlearn/parameter_learning.py module, line 155.

Thank you very much. Best regards.

**Comment by erdogant** [comment]
Thank you for notifying me. I pass the verbose into the function and it should be solved in the next release!

**Comment by erdogant** [comment]
The update is here!
update with: `pip install -U bnlearn`

**Comment by paulaten** [comment]
Thank you!

I hope this message finds you well. I wanted to thank you for the latest version of bnlearn library, it has been very useful for my work. However, I noticed that there is still an issue with a line of code that I previously mentioned.

I just ran the latest version (0.7.15) and the message still appears even though I set verbose=0 in bnlearn/structure_learning.fit (line 192). Is there something else that I need to do in order to resolve this issue?

Thank you for your help and for all the hard work you put into maintaining this library.

Best regards.
